keyboards/gboards/dicts/aset/_generator/main.go

```diff
@@ -0,0 +1,298 @@
+package main
+
+import (
+	"io/ioutil"
+	"fmt"
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"strings"
+	"sort"
+	"hash/crc64"
+	//"encoding/base64"
```

Comment:
Can this be removed?

Reply:
I'd prefer to keep it in, makes it easy for people to regenerate the default dictionaries if needed.